nginx/nginx.conf

```
@@ -0,0 +1,24 @@
+worker_processes 4;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    server {
+        listen 80;
+        root  /usr/share/nginx/html;
+        include /etc/nginx/mime.types;
+
+        # route all other requests to the index for react router
+        location / {
+            try_files $uri /index.html;
+        }
+
+        # airports, flights and positions uris are passed to backend
+        location ~ ^/(api) {
+            proxy_pass http://fids-backend:5000;
```

```suggestion
            proxy_pass http://fids-backend:5000/;
```
I could be wrong, but I _think_ that by specifying this trailing slash, you can remove all the `/api/` prefixes you had to add to the routes in the backend. I think it's cleaner that way.

If this doesn't work like I thought, there is likely some other way to accomplish this.

---

Yeah, this works as expected and makes the backend much cleaner